### DISCUSSION TITLE
Eclipse JNoSQL Version 1.1.0 adds support for Oracle NoSQL Database

### DISCUSSION THREAD

**Discussion by dario-vega** [oracle/nosql#12] [Announcements]
📢 📢 📢 We are excited to announce that  Eclipse JNoSQL Version 1.1.0 adds support for Oracle NoSQL Database

### Eclipse JNoSQL

**Eclipse JNoSQL** is a Java-based framework that facilitates seamless integration between Java applications and nosql databases. It leverages Java enterprise standards, specifically **Jakarta NoSQL** and **Jakarta Data**, to simplify working with NoSQL databases. The primary objective of this framework is to reduce the ramp-up associated with using NoSQL databases while harnessing the full power of **Jakarta EE** and **Eclipse MicroProfile**.

With Eclipse JNoSQL, developers can easily integrate NoSQL databases into their projects using Widfly, Quarkus, Helidon, or other Java platforms. This framework bridges the Java application layer and various NoSQL databases, making it easier to work with these databases without having a dive-deep knowledge of them. The beauty of this framework is that it can work with many NoSQL databases, including our favorite Oracle NoSQL Database.

### Getting Started

I recommend the following blog and the following article
- https://blogs.oracle.com/nosql/post/getting-started-accessing-oracle-nosql-database-using-jakarta-nosql
- https://dzone.com/articles/building-a-rest-application-with-oracle-nosql-usin

### Want to learn more
I recommend the following links
- https://github.com/eclipse/jnosql-databases
- http://www.jnosql.org/
- https://github.com/JNOSQL/demos-se

**Comment by dario-vega** [comment]
@otaviojava thanks for your help

**Reply by otaviojava** [reply]
That is my pleasure @dario-vega

**Reply by dario-vega** [reply]
@otaviojava what's new in Version 1.1.1?

**Reply by otaviojava** [reply]
@dario-vega, we did a lot of enhancements and more integration with Jakarta Data.

You can check the whole information here with a sample using Oracle NoSQL:

https://dzone.com/articles/understanding-nosql-databases-with-java

**Comment by dario-vega** [comment]
@otaviojava I think your latest blog is https://dzone.com/articles/nosql-database-integration-in-java-with-eclipse-jnosql-1-1-3

**Reply by otaviojava** [reply]
Yes, this is the one.
